extensions/2.0/Khronos/KHR_texture_basisu/README.md

```diff
@@ -0,0 +1,173 @@
+
+To cover a broad range of use cases, this extension allows two Basis Universal formats: ETC1S with BasisLZ and UASTC with optional Zstandard compression.
+
+- KTX header fields for ETC1S with BasisLZ payloads
+  - `supercompressionScheme` must be `1` (BasisLZ).
+  - DFD `colorModel` must be `KHR_DF_MODEL_ETC1S`.
+  - RGB textures must use a single DFD channel with value `KHR_DF_CHANNEL_ETC1S_RGB`.
+  - RGBA textures must use two DFD channels with values `KHR_DF_CHANNEL_ETC1S_RGB` and `KHR_DF_CHANNEL_ETC1S_AAA`.
+
+- KTX header fields for UASTC payloads
+  - `supercompressionScheme` must be `0` (None) or `2` (Zstandard).
+  - DFD `colorModel` must be `KHR_DF_MODEL_UASTC`.
+  - RGB textures must use a single DFD channel with value `KHR_DF_CHANNEL_UASTC_RGB`.
+  - RGBA textures must use a single DFD channel with value `KHR_DF_CHANNEL_UASTC_RGBA`.
+
+Regardless of the format used, these additional restrictions apply for compatibility reasons:
+
+- Swizzling metadata (`KTXswizzle`) must be `rgba` or omitted.
+- Orientation metadata (`KTXorientation`) must be `rd` or omitted.
+- Colorspace information in the DFD must match the expected usage, namely:
+  - For textures with color data, `colorPrimaries` must be set to `KHR_DF_PRIMARIES_BT709`; `transferFunction` to `KHR_DF_TRANSFER_SRGB`.
+  - For textures with non-color data, `colorPrimaries` must be set to `KHR_DF_PRIMARIES_UNSPECIFIED`; `transferFunction` to `KHR_DF_TRANSFER_LINEAR`.
+- `KHR_DF_FLAG_ALPHA_PREMULTIPLIED` flag must not be set unless the material's specification requires premultiplied alpha.
+- When a texture refers to a sampler with mipmap minification or when the sampler is undefined, the KTX image should contain a full mip pyramid.
+  > **Implementation Note**: Depending on the platform features available, engines may need to decompress the texture and generate missing mipmaps at runtime when a full mip pyramid is not available.
+- `pixelWidth` and `pixelHeight` must be multiples of 4.
+  > **Implementation Note**: Exporters may opt to make the dimensions powers of 2. Combined with all mip levels present this would ensure maximum compatibility.
+
+### Using KTX v2 Images with Basis Universal Supercompression for Material Textures
+
+When a texture referencing a KTX v2 image with Basis Universal supercompression is used for glTF 2.0 material maps (such as base color), the KTX v2 image must be of **2D** type as defined in the KTX v2 Specification, Section 4.1.
+
+Exporters should choose the most appropriate universal texture format based on the texture's usage and content. As a general rule, textures with color data should use ETC1S while textures with non-color data (such as roughness-metallic or normal maps) should use UASTC.
+
+When the high-quality GPU formats (ASTC or BC7) are not available at runtime, engines should transcode UASTC textures with non-color data to uncompressed RGBA to avoid artifacts.
+
+## Known Implementations
+
+None.
```

Comment:
Should we populate this?

Reply:
Starting point in #1771

Reply:
@donmccurdy 
Should we resolve that issue before adding anything here?

Reply:
I think that issue is going to track a lot more things than we want to list here, and ultimately the list should end up on the glTF Project Explorer.

In the meantime, I think I would just list 1-2 authoring tools and 1-2 client tools in the extension, which we can do now:

Authoring:

- [gltfpack](https://github.com/zeux/meshoptimizer)
- [glTF-Transform CLI](https://gltf-transform.donmccurdy.com/cli.html)

Viewing: 

- [Babylon](https://www.babylonjs.com/)
- [three.js](https://threejs.org/)

Obviously libktx and Universal-Texture-Transcoders are important for building an implementation, but we can't really claim that they "implement" this extension.  Better to follow up with a separate developer guide?